org.videolan.VLC.json

```diff
@@ -0,0 +1,867 @@
+        }
+      ]
+    },
+    {
+      "name": "libebml",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dl.matroska.org/downloads/libebml/libebml-1.3.4.tar.bz2",
+          "sha256": "c50d3ecf133742c6549c0669c3873f968e11a365a5ba17b2f4dc339bbe51f387"
+        }
+      ]
+    },
+    {
+      "name": "libmatroska",
+      "config-opts": [
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dl.matroska.org/downloads/libmatroska/libmatroska-1.4.5.tar.bz2",
+          "sha256": "79023fa46901e5562b27d93a9dd168278fa101361d7fd11a35e84e58e11557bc"
+        }
+      ]
+    },
+    {
+      "name": "libcdio",
+      "config-opts": [
+        "--disable-vcd-info",
+        "--enable-cpp-progs",
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ftp.gnu.org/gnu/libcdio/libcdio-0.83.tar.gz",
+          "sha256": "235017e3eccb86424f9c108f2c5d5fca62630bda8c9dcf23b425ba9c5e2482c0"
+        },
+        {
+          "type": "patch",
+          "path": "libcdio.patch"
+        }
+      ]
+    },
+    {
+      "name": "libssh2",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.libssh2.org/download/libssh2-1.8.0.tar.gz",
+          "sha256": "39f34e2f6835f4b992cafe8625073a88e5a28ba78f83e8099610a7b3af4676d4"
+        }
+      ]
+    },
+    {
+      "name": "libvncserver",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/LibVNC/libvncserver/archive/LibVNCServer-0.9.11.tar.gz",
+          "sha256": "193d630372722a532136fd25c5326b2ca1a636cbb8bf9bb115ef869c804d2894"
+        }
+      ]
+    },
+    {
+      "name": "libdvdread",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ftp.videolan.org/videolan/libdvdread/5.0.3/libdvdread-5.0.3.tar.bz2",
+          "sha256": "321cdf2dbdc83c96572bc583cd27d8c660ddb540ff16672ecb28607d018ed82b"
+        }
+      ]
+    },
+    {
+      "name": "libdvdnav",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ftp.videolan.org/videolan/libdvdnav/5.0.3/libdvdnav-5.0.3.tar.bz2",
+          "sha256": "5097023e3d2b36944c763f1df707ee06b19dc639b2b68fb30113a5f2cbf60b6d"
+        }
+      ]
+    },
+    {
+      "name": "libmodplug",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://downloads.sourceforge.net/modplug-xmms/libmodplug-0.8.8.5.tar.gz",
+          "sha256": "77462d12ee99476c8645cb5511363e3906b88b33a6b54362b4dbc0f39aa2daad"
+        }
+      ]
+    },
+    {
+      "name": "ffmpeg",
+      "config-opts": [
+        "--enable-pic",
+        "--enable-shared",
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ffmpeg.org/releases/ffmpeg-2.8.9.tar.xz",
```

Comment:
This version is fairly old, I know there are patches around for building with the latest ffmpeg.

Also the default configuration builds a ton of unused things like cli tools and docs.